### ISSUE TITLE
mAP for ssd and ssdlite caffe models without any finetuning?

### ISSUE THREAD

**Issue by fcakir** [chuanqi305/MobileNetv2-SSDLite#63]
Does the repo owner or any others have accuracy values for the ssd and ssdlite caffe models with the tensorflow weights? 
The mAP I get with ssd mobilenetv2 is 0.023, which is significantly lower than the official result. This is even when using relu6. 
Has anyone successfully reproduced the official mAP values?
